src/uwuw/uwuw.cpp

```diff
@@ -64,7 +65,11 @@ std::string UWUW::get_full_filepath(std::string filename) {
   filename.erase(std::remove(filename.begin(), filename.end(), ' '),
                  filename.end());
   // use stdlib call
+#ifndef _WIN32
   const char* full_filepath = realpath(filename.c_str(), NULL);
+#else
+  const char* full_filepath = filename.c_str();
```

Comment:
Still interested in getting uniform behavior here...

Reply:
I still don't know what to do for this one....

Reply:
let's leave it for now

Reply:
Although the simple option is to have all platforms behave like the Windows one.  It is more limiting, but consistent

Reply:
Other than the problems caused by this in PyNE, wouldn't `filesystem::canonical()` work here?

Reply:
yes I think so